database/postgres/composition.yaml

```diff
@@ -1,39 +1,39 @@
 apiVersion: apiextensions.crossplane.io/v1
 kind: Composition
 metadata:
-  name: xpostgresqlinstances.aws.platformref.crossplane.io
+  name: xpostgresqlinstances.aws.platformref.upbound.io
   labels:
     provider: aws
 spec:
   writeConnectionSecretsToNamespace: upbound-system
   compositeTypeRef:
-    apiVersion: aws.platformref.crossplane.io/v1alpha1
+    apiVersion: aws.platformref.upbound.io/v1alpha1
     kind: XPostgreSQLInstance
   resources:
     - base:
-        apiVersion: database.aws.crossplane.io/v1beta1
-        kind: DBSubnetGroup
+        apiVersion: rds.aws.upbound.io/v1beta1
+        kind: SubnetGroup
         spec:
           forProvider:
             region: us-west-2
             description: An excellent formation of subnetworks.
           deletionPolicy: Delete
       name: compositePostgreSQLInstanceDbSubnetGroup
       patches:
-        - fromFieldPath: "spec.parameters.clusterRef.id"
-          toFieldPath: spec.forProvider.subnetIdSelector.matchLabels[networks.aws.platformref.crossplane.io/network-id]
+        - fromFieldPath: spec.parameters.clusterRef.id
+          toFieldPath: spec.forProvider.subnetIdSelector.matchLabels[networks.aws.platformref.upbound.io/network-id]
     - base:
-        apiVersion: database.aws.crossplane.io/v1beta1
-        kind: RDSInstance
+        apiVersion: rds.aws.upbound.io/v1beta1
+        kind: Instance
         spec:
           forProvider:
             region: us-west-2
             dbSubnetGroupNameSelector:
               matchControllerRef: true
-            dbInstanceClass: db.t2.small
-            masterUsername: masteruser
+            instanceClass: db.t3.micro
+            username: masteruser
             engine: postgres
-            engineVersion: "9.6"
+            engineVersion: "13.7"
             skipFinalSnapshotBeforeDeletion: true
```

Comment:
Getting a   `destroy failed: final_snapshot_identifier is required when skip_final_snapshot is false:`, when trying to delete.